docs/source/cognite.rst

```diff
@@ -768,3 +768,62 @@ Convert timestamp to milliseconds since epoch
 Convert milliseconds since epoch to datetime
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 .. autofunction:: cognite.client.utils.ms_to_datetime
+
+
+Experimental features
+=====================
+These features are not part of the core v1.0.0 API, and subject to breaking changes.
```

Comment:
This could cause some confusion with the `cognite-sdk-core` distribution. Maybe something like "These features are subject to breaking changes and should not be used in any production code." instead.